packages/ai/util/get-effective-abort-signal.ts

```diff
@@ -0,0 +1,26 @@
+export function getEffectiveAbortSignal(
```

Comment:
is this file still needed?

Reply:
My bad. Didnt push the change. Its done now.